docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -678,9 +689,120 @@ This could be done by assigning unique (or practically unique, with very low pro
 2.3.6 Component 10: International Emigration (Out-Migration)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Simulants may leave the US to live in other countries.
+As with immigration, there are three types of emigration events that can occur:
+
+#. Household moves, when an entire household moves out of the US as a unit.
+#. GQ person moves, when a GQ person moves out of the US individually.
+#. Non-reference-person moves, when a single non-GQ person leaves their household to move out of the US.
+
+Data sources and analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use the Net International Migration (NIM) estimates from the Census
+Bureau's Population and Housing Unit Estimates (PopEst) program to determine the
+number of emigrants per year. [Census_PopEst]_
+Specifically, we use the 2018-2019 annual estimates, in the assumption that this
+(pre-COVID) year's emigration can be applied to each future year in the simulation.
+
+We subtract out immigration, which we estimate from the ACS PUMS'
+migration question as described in the previous section, to isolate emigration.
+Specifically, these three quantities are related by the equation
+:math:`\text{NIM} = \text{immigration} - \text{emigration}`.
+
+The NIM estimates are made by the PopEst team by combining information
+about immigration from ACS with information about emigration from demographic analysis
+(for those born outside the US) and analysis of foreign censuses (for those born in
+the US). [Census_PopEst_Methodology]_
+Without access to the source data, we cannot replicate these methods, which is why we
+use the published NIM values instead of directly estimating emigration.
+
+The NIM values are not published fully stratified.
+Out of the available stratifications, we chose to use the values stratified
+by (broad categories of) race/ethnicity, because these are most likely to have
+PRL implications.
+
+Inspired by the methodology of the PopEst team at the Census Bureau,
+we further stratify emigration by assuming that **emigrants** have the same
+characteristics as **immigrants**.
```

Comment:
This is interesting to me and not intuitive. I would imagine that different folks might want to enter and exit the US, but I assume the Census did this for a review!

Reply:
Yes, it's not necessarily intuitive. The PopEst team does this in their own modeling with essentially no justification -- they just state that it is an assumption. (In fact, it is a bit more complicated than even that, because for people born in the US they use the set of immigrants *or* between-state movers which seems like an even bigger leap to me.)